CommonLibSF/include/RE/A/AIProcess.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "ActorPackage.h"
+
+namespace RE
+{
+	struct MiddleLowProcessData;
+	struct MiddleHighProcessData;
+	struct HighProcessData;
+
+	// F8
+	class AIProcess
+	{
```

Comment:
```suggestion
    class AIProcess
    {
    public:
```
Without this members in a class are private by default

Reply:
Oh if we're doing this there's a ton of other places in the codebase where redundant access modifiers (and a lot of other redundant stuff) should be removed too

Reply:
It's adding public in this case